test/testinfra.py

```diff
@@ -193,6 +175,22 @@ def __init__(self, base="/repos/cockpit-project/cockpit/"):
                 raise
         self.available = self.token and True or False
 
+        # Try to load the whitelists
```

Comment:
Why did you move this code out of the function? It's more than a few lines and performs a specific task.